ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -26,26 +26,21 @@ class SegmentPillButton: UIButton {
         }
     }
 
-    var tokens: SegmentedControlTokens {
-        didSet {
-            updateTokenizedValues()
-            updateUnreadDot()
-        }
-    }
+    var tokenSet: SegmentedControlTokenSet
 
-    private func updateTokenizedValues() {
-        titleLabel?.font = UIFont.fluent(tokens.font, shouldScale: false)
-        let verticalInset = tokens.verticalInset
-        let horizontalInset = tokens.horizontalInset
+    func updateTokenizedValues() {
```

Comment:
Why the switch to internal? Also, is this still getting called? I don't see it anywhere in SegmentedControl.swift, so are the actual buttons of the segmented control getting updates? Unless we call this from the SegmentedControl, I don't think the font or insets will get updates (though, the insets need a completely different update, since they're getting deprecated...)

Reply:
Yep you're right, fixing now